tests/plugins/test_terminator.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+"""
+Test Terminator
+---------------
+"""
+
+from jinja2 import Template
+
+import folium
+from folium import plugins
+
+def test_terminator():
+    m = folium.Map([45., 3.], zoom_start=1)
```

Comment:
Appart from this, I've applied you comments (spaces), and rebased.
Ready for merge.
Thanks @ocefpaf